base/compiler/abstractinterpretation.jl

```diff
@@ -1062,7 +1062,7 @@ function const_prop_methodinstance_heuristic(
     # was able to cut it down to something simple (inlineable in particular).
     # If so, there's a good chance we might be able to const prop all the way
     # through and learn something new.
-    if isdefined(method, :source) && ccall(:jl_ir_flag_inlineable, Bool, (Any,), method.source)
+    if isdefined(method, :source) && ccall(:jl_ir_flag_inlineable, Bool, (Any,), method.source) != 0x0000
```

Comment:
`Bool -> UInt16`?

Reply:
Ah yeah, silly mistake, thanks for the catch.